packages/dds/tree/src/test/feature-libraries/sequence-field/sequenceFieldEditor.spec.ts

```diff
@@ -13,30 +13,32 @@ import { TestChangeset, MarkMaker as Mark } from "./testEdits.js";
 
 const id: ChangesetLocalId = brand(0);
 
-describe("SequenceField - Editor", () => {
-	it("child change", () => {
-		const childChange = TestChange.mint([0], 1);
-		deepFreeze(childChange);
-		const actual = SF.sequenceFieldEditor.buildChildChange(42, childChange);
-		const expected: TestChangeset = [{ count: 42 }, Mark.modify(childChange)];
-		assert.deepEqual(actual, expected);
-	});
+export function testEditor() {
```

Comment:
should these still be spec files now that they're not actually running any tests?

Reply:
I've now changed them to .test.ts